lib/fathead/spring-framework/fetch_docs.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python
```

Comment:
nit: please rename to `fetch.py` -- that's the canonical name for the fetching script 👍